server/index.js

```diff
@@ -1,9 +1,13 @@
 const path = require('path')
-require('dotenv').config({path: path.join(__dirname,'.env')})
+require('dotenv').config(path.join(__dirname, '.env'))
```

Comment:
This should be set up the was that master is set up. See https://www.npmjs.com/package/dotenv#path